packages/studio-base/src/components/ExperimentalFeatureSettings.tsx

```diff
@@ -60,6 +60,13 @@ const features: Feature[] = [
     name: "Plot panel series in settings",
     description: <>Allow editing plot panel data series in the sidebar.</>,
   },
+  {
+    key: AppSetting.ENABLE_NEW_UI,
+    name: "Enable new user interface",
+    description: (
+      <>Try out redesigned navigation and layout. Warning: We’re still figuring it our</>
+    ),
```

Comment:
Let's be a bit more specific
```suggestion
    key: AppSetting.ENABLE_NEW_TOPNAV,
    name: "Enable new top navigation",
    description: (
      <>Test our redesigned navigation bar.</>
    ),
```

Reply:
I do think that this flag will eventually catch the sidenav removal as well. So perhap something more like "Try our redesigned navigation"